viewer/packages/360-images/src/annotation-styling/AnnotationStyleTracker.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { AnnotationModel } from '@cognite/sdk/dist/src';
+import { ImageAnnotationObject } from '../annotation/ImageAnnotationObject';
+import { Image360AnnotationAppearance, Image360AnnotationAppearanceEdit, Image360AnnotationFilter } from './types';
+
+import { Color, MeshBasicMaterial } from 'three';
+
+export class AnnotationStyleTracker {
+  private readonly _annotationStyleAssignments: [Image360AnnotationFilter, Image360AnnotationAppearanceEdit][] = [];
```

Comment:
Maybe this should be a map? Just seems more ergonomic based on how the collection is used.

Reply:
Outdated, removed `AnnotationStyleTracker`